tests/queries/tests.py

```diff
@@ -2104,6 +2104,13 @@ def test_sliced_delete(self):
         DumbCategory.objects.filter(id__in=DumbCategory.objects.order_by('-id')[1:]).delete()
         self.assertEqual(set(DumbCategory.objects.values_list('id', flat=True)), {3})
 
+    def test_subquery_different_database(self):
+        # Error Checking subquery to generate a different database
+        with self.assertRaisesMessage(ValueError,
```

Comment:
Use this style to avoid a flake8 warning (currently ignored but we might fix it at some point):
```
with self.assertRaisesMessage(
    ValueError,
    "Subqueries aren't allowed across different databases. Instead, "
    "force the inner query to be evaluated using `list(inner_query)`."
):
```